src/components/styles/Book.module.css

```diff
@@ -0,0 +1,154 @@
+.book {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  width: 100%;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  margin-top: 10px;
+}
+
+.analytics > div:nth-child(2) button {
+  margin-top: 20px;
+  width: 11.5rem;
+  height: 2.063rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border: none;
+}
+
+.book > div:nth-child(1) ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.book > div:nth-child(1) ul li button {
+  border: none;
+  background-color: white;
+}
```

Comment:
- [ ] I would suggest you to make sure the nav menu to be align with the addbook buyyon as shown on the given design as you can see below you design is different to the original design please kindly stick to the given design as much as possible for user friendly.
- [ ] given design
![bookstore](https://github.com/nilab-khwaja/bookstore/assets/92458236/8e49edb4-4328-4505-92cb-4bfb9b3b9c03)

- [ ] your design
![Screenshot 2023-09-28 191307](https://github.com/nilab-khwaja/bookstore/assets/92458236/012b0a7c-7768-48f8-924b-dbfa8bc44aca)